apps/server/src/services/ElasticSearchClient.ts

```diff
@@ -48,24 +49,39 @@ export async function reIndexElasticSearch() {
   } catch {
```

Comment:
Line 45 don't we already know there isn't an institutions index, because we checked on line 34?

Reply:
Yeah, I think originally that reIndex function was meant to be re-used but then probably got refactored and didn't get cleaned up.